Bottom: 5eba6bdf5055a343ca6015f657f691c43a1d53a9
Top:    671bb05e7c554f5add4b9144e9654acc923c2433
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-14 14:49:52 -0500

Refresh of all-distribute-image-repo-checkout

---
diff --git a/roles/docker-gitlab/tasks/main.yml b/roles/docker-gitlab/tasks/main.yml
index aeab7bf9..dfeed36f 100644
--- a/roles/docker-gitlab/tasks/main.yml
+++ b/roles/docker-gitlab/tasks/main.yml
@@ -58,12 +58,6 @@
     group: tape
     mode: 'u=rw,g=r,o='
 
-- name: copy env-file
-  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-gitlab/gitlab.env.list {{ docker_restore_config_base_dir }}/{{ gitlab_dv_name }}'
-
-- name: copy script
-  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-gitlab/gitlab.sh {{ docker_restore_config_base_dir }}/{{ gitlab_dv_name }}'
-
 - name: exists - state file
   stat:
     path: '{{ docker_restore_config_base_dir }}/{{ gitlab_dv_name }}/restore.date.txt'
@@ -80,6 +74,12 @@
     version: master
     dest: '{{ docker_projects_dir }}/docker-gitlab'
 
+- name: copy env-file
+  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-gitlab/gitlab.env.list {{ docker_restore_config_base_dir }}/{{ gitlab_dv_name }}'
+
+- name: copy script
+  command: 'cp --verbose --update --backup=numbered {{ docker_projects_dir }}/docker-gitlab/gitlab.sh {{ docker_restore_config_base_dir }}/{{ gitlab_dv_name }}'
+
 - name: build image
   docker_build:
     image_name: '{{ gitlab_image_name }}'
